web/frontend/src/lib/components/collapse.svelte

```diff
@@ -39,11 +66,12 @@ onMount(() => {
 </script>
 
 <div class="relative w-full">
-  <div
+  <button
```

Comment:
This looks like it creates a `<button>` inside another `<button>`, which is against HTML spec